tests/test-lib/registry.ts

```diff
@@ -50,9 +42,6 @@ app.use('/v2/', (req, res) => {
 		return res.status(403).json(ERROR_DENIED);
 	}
 
-	// https://docs.docker.com/registry/spec/auth/jwt/
-	const jwt = jsonwebtoken.decode(auth.token as string) as { access: Access[] };
```

Comment:
I don't know if this was intended to be used @klutchell ?

Reply:
It's been a long time since I looked at these tests, but I can walk through it locally and confirm.

Perhaps it was meant to be used on line 52?

Reply:
IIRC it was just to ensure we got a valid token as part of the request, and that my other tests weren't passing nonsense. I definitely didn't need to save the return value unless I missed deleting it with legacy code. It's been so long and I failed to comment it.